src/Plutus/Certification/API/Routes.hs

```diff
@@ -60,6 +60,7 @@ type AbortRunRoute = "run"
   :> Description "Abort a run"
   :> AuthProtect "public-key"
   :> Capture "id" RunIDV1
+  :> QueryParam "delete" Bool
```

Comment:
Shouldn't we also have a separate `DELETE` route?

Reply:
At first, that was my choice too. But a separate route would also have to abort the run if it is still running besides deleting it from history. 
So in talking with @RSoulatIOHK it was suggested to use the same route and not add another one.
And because someone might want just to abort the run, but to keep it in history, this is how the optional query param appeared